src/app/Providers/components/AddEditProviderModal/AddEditProviderModal.tsx

```diff
@@ -214,16 +216,15 @@ const AddEditProviderModal: React.FunctionComponent<IAddEditProviderModalProps>
                   isRequired
                   fieldId="vmware-username"
                 />
-                <ValidatedTextInput
+                <ValidatedPasswordInput
                   field={vmwareForm.fields.password}
-                  type="password"
                   label="Password"
                   isRequired
                   fieldId="vmware-password"
                 />
                 <ValidatedTextInput
                   field={vmwareForm.fields.fingerprint}
-                  label="Certificate SHA1 Fingerprint"
+                  label="SHA-1 Fingerprint"
```

Comment:
"Fingerprint" should be "fingerprint", per PF4 guidelines.

Reply:
Agh dammit, thanks @apinnick. Will fix in another PR.